### ISSUE TITLE
Is bert network weight update while training?

### ISSUE THREAD

**Issue by LMC63** [tczhangzhi/torch-text-similarity#4]
hello @tczhangzhi, it's a useful repo and I have read your code. I notice that the network is made up of bert layer and regression layer and I want to know If the bert layer weight update while doing Supervised learning using data as below:

            "sen1": "有图标 但是开通不了 怎么回事",
            "sen2": "都正确开通不了是怎么回事",
            "label": "1"

Or only weight of regression layer will update, could you tell me? Thanks.

**Comment by tczhangzhi** [comment]
All weights will be updated.

If you want to update only the regression layer, you can set the `requires_grad` of other parts to `False`. You may need a larger Chinese corpus and pretrained weights, for example, https://github.com/IAdmireu/ChineseSTS and `model_name=bert-base-chinese`.

**Comment by tczhangzhi** [comment]
Due to working on other projects, I may be slow to reply and fix bugs. 

If you have any new progress, feel free to pull the request.

**Comment by LMC63** [comment]
Thanks for your kindly reply!
